lib/features/support/app-automate.rb

```diff
@@ -0,0 +1,63 @@
+require 'appium_lib'
```

Comment:
I would expect this to be added to deps?

Reply:
It's already been added previously